gremlin-server/src/main/bin/gremlin-server.sh

```diff
@@ -96,7 +96,7 @@ fi
 
 # Build Java CLASSPATH
 CP="$GREMLIN_HOME/conf/"
-CP="$CP":$( echo $GREMLIN_HOME/lib/*.jar . | sed 's/ /:/g')
+CP="$CP":$( find "$GREMLIN_HOME/lib" -name "*.jar" -print | tr '\n' ':' )
```

Comment:
There's a slight difference in the output of these 2 commands.

`echo $GREMLIN_HOME/lib/*.jar . | sed 's/ /:/g'` produces something like this:
`path_to_x.jar:path_to_y.jar:path_to_z.jar:.`

whereas `find "$GREMLIN_HOME/lib" -name "*.jar" -print | tr '\n' ':'` produces something like this:
`path_to_x.jar:path_to_y.jar:path_to_z.jar:`

The new classpath is missing the current directory (`.`).

Also just a small nit, the new classpath is ending with a trailing `:`, I don't think this causes any issues but it is a bit atypical.